app/uk/gov/hmrc/soletraderidentificationfrontend/services/SoleTraderIdentificationService.scala

```diff
@@ -161,7 +161,7 @@ object SoleTraderIdentificationService {
   val NinoKey = "nino"
   val SautrKey = "sautr"
   val DateOfBirthKey = "dateOfBirth"
-  val IdentifiersMatchKey: String = "identifiersMatch"
+  val identifiersMatchKey: String = "identifiersMatch"
```

Comment:
I would not have changed this as all of the key names start with a capital. However, it still works.